pkg/api/errors/validation.go

```diff
@@ -84,13 +84,29 @@ type ValidationError struct {
 var _ error = &ValidationError{}
 
 func (v *ValidationError) Error() string {
-	s := spew.Sprintf("%s: %s '%+v'", v.Field, v.Type, v.BadValue)
-	if v.Detail != "" {
+	var s string
+	if v.Type == ValidationErrorTypeRequired && isEmpty(v.BadValue) {
```

Comment:
I have had a long-standing desire to actually just remove the value option to NewFieldRequired() - in EVRY SINGLE case* it is verified to be empty before we pass it in anyway.  Let's just get rid of it.

* pkg/config/config.go passes the error value there - this is just WRONG